CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Pool] Add tooltip for `no support` icon (PR [#6505](https://github.com/vatesfr/xen-orchestra/pull/6505))
```

Comment:
Keep the changelog entries user oriented as much as possible and make sure anyone can understand what it means without context:
```suggestion
- [Pool] Add tooltip on "no XCP-ng Pro support" warning icon (PR [#6505](https://github.com/vatesfr/xen-orchestra/pull/6505))
```